Lib/test/test_tomllib/test_error.py

```diff
@@ -39,6 +39,15 @@ def test_invalid_char_quotes(self):
             tomllib.loads("v = '\n'")
         self.assertTrue(" '\\n' " in str(exc_info.exception))
 
+    def test_type_error(self):
+        with self.assertRaises(TypeError) as exc_info:
+            tomllib.loads(b"v = 1")  # type: ignore[arg-type]
```

Comment:
Do we run mypy on this file?

Reply:
I don't think so. Tomli does, though, and all tests were initially directly copied from there. These were left in to make maintenance of the backport easier. That's why you'll find a `type: ignore` e.g. [here](https://github.com/python/cpython/blob/0387c34f7c91428681ca8a4ba4e3d22b9acffde4/Lib/test/test_tomllib/test_misc.py#L37) already.

Reply:
Should we keep it then? (if there is nothing to gain from a CPython side I would just remove it).

Reply:
Probably not my decision to make. But according to the logic used then, I think it should stay, and if not then these (and any `# noqa`s) should probably be deleted throughout the tomllib library and tests.